kite-data/kite-data-core/src/main/java/org/kitesdk/data/spi/filesystem/CSVAppender.java

```diff
@@ -81,7 +81,6 @@ public void flush() throws IOException {
   @Override
   public void sync() throws IOException {
     flush();
-    Hadoop.FSDataOutputStream.hsync.invoke(outgoing);
```

Comment:
Why are we removing the call to `hsync()`?

`hflush()` doesn't guarantee that data has been written to stable storage.